gdal/ogr/ogrsf_frmts/flatgeobuf/ogrflatgeobuflayer.cpp

```diff
@@ -530,6 +536,7 @@ OGRErr OGRFlatGeobufLayer::parseFeature(OGRFeature *poFeature, OGRGeometry **ogr
         m_offset += m_featureSize + sizeof(uoffset_t);
         fid = m_featuresPos;
     } else {
+        //m_offset = m_offsetInit;
```

Comment:
Is this an actual comment or it's been overlooked?

Reply:
Left over from bug hunting iteration, should be removed.

Reply:
I see. I thought it's legit and for a moment I even thought the comment was supposed to read
```
 //m_offset == m_offsetInit;
```
what would assume a documenting `assert(m_offset == m_offsetInit);`